src/components/map/Layers/Countries.tsx

```diff
@@ -118,11 +118,11 @@ const Countries = (map: mapboxgl.Map | undefined, {estimateGradePrevalences, cou
                         );
                     }    
                     
+                    setHighlight(e.features[0].id)
```

Comment:
any reason why this was moved up?

Reply:
to create better Feng shui within the code